### DISCUSSION TITLE
Disabling default global shortcuts

### DISCUSSION THREAD

**Discussion by alaestor** [nix-community/plasma-manager#416] [Q&A]
Is there (or, could there be) a concise way to disable default shortcuts? I'm aware of some [stack exchange questions](https://unix.stackexchange.com/questions/576633/how-to-disable-all-global-hotkeys-in-kde-plasma) that have covered this in other contexts, but I'd like to only have the global shortcuts that I declare in my .nix without having to also disable everything, as that's quite noisy and a pain to manage.

**Comment by HeitorAugustoLN** [comment]
I think the best way to handle this in this case would be running this script and getting the shortcuts from rc2nix running `nix run github:nix-community/plasma-manager`
